web/beacon-app/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Adding this should make it so that we don't see errors under `cy` when opening the test files.